mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -135,48 +206,65 @@ export class CompareRunView extends Component {
             </h1>
           }
         >
-          <div className='responsive-table-container'>
+          <div className='responsive-table-container' ref={this.tableContainerRef}>
             <table className='compare-table table'>
-              <thead>
+              <thead
+                className='table-block no-scrollbar'
+                style={tableBlockWidthStyle}
+                onScroll={this.onTableBlockScrollHandler}
+              >
                 <tr>
-                  <th scope='row' className='row-header'>
+                  <th scope='row' className='head-value sticky_header' style={colWidthStyle}>
                     <FormattedMessage
                       defaultMessage='Run ID:'
                       description='Row title for the run id on the experiment compare runs page'
                     />
                   </th>
                   {this.props.runInfos.map((r) => (
-                    <th scope='column' className='data-value' key={r.run_uuid}>
-                      <Link to={Routes.getRunPageRoute(r.getExperimentId(), r.getRunUuid())}>
-                        {r.getRunUuid()}
-                      </Link>
+                    <th scope='row' className='data-value' key={r.run_uuid} style={colWidthStyle}>
+                      <Tooltip
```

Comment:
Instead of `Tooltip`, can we use [the title attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title)?

https://user-images.githubusercontent.com/17039389/151673078-4bd6e8ed-9198-41c4-b41e-d2553e11e430.mov

Reply:
`Tooltip` always instantly shows up when you hover on the cell, but I feel like this behavior unnecessarily draws the user's attention, especially when the cell content is NOT truncated.

Reply:
Does  'title' support only display when truncated ?

Reply:
@dbczumar 
Do you prefer use "title" or "Tooltip" ?

Reply:
i remember corey prefer "blue" background for the hovor text. So I keep tooltip. But I increase the mouseEnterDelay to be 1.0 second (i.e. only mouse hover above 1 second, then the tooltip will show)